x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/action/user/ChangePasswordRequestBuilderTests.java

```diff
@@ -75,39 +76,8 @@ public void testWithHashedPasswordWithDifferentAlgo() throws IOException {
         assertThat(request.passwordHash(), equalTo(hash));
     }
 
-    public void testWithHashedPasswordAndNoopSystemHasher() throws IOException {
```

Comment:
These tests were testing unsupported case. I got this wrong the first time. The `NOOP`(clear text) hasher cannot be configured as the password hashing algorithm. It can only be configured as a password caching algorithm.